.github/workflows/generate-packages.yml

```
@@ -0,0 +1,193 @@
+    name: Determine Packages to Generate
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      # Only checkout for push events (need to check changed paths)
+      # Other triggers don't need checkout
+      - name: Checkout (only for push events)
+        if: github.event_name == 'push'
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1 # Shallow clone - only need current commit for path checking
+    outputs:
```

actionlint (1.7.9)

could not parse as YAML: did not find expected key

(syntax-check)